service.subtitles.legendastv/addon.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+   id="service.subtitles.legendastv"
+   name="Legendas.TV"
+   version="2.4.4"
+   provider-name="gfjardim">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="vfs.libarchive" version="1.0.5" />
+  </requires>
+  <extension point="xbmc.subtitle.module" library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Legendas.TV</summary>
+    <description lang="en">Search and download subtitles from Legendas.TV</description>
+    <description lang="pt">Procura e baixa legendas de Legendas.TV</description>
```

Comment:
pt_PT

Reply:
Would pt_BR also be ok? I'm Brazilian and so is the original author, IIRC, so this likely would follow our spelling and vocabulary expectations more then what's used in pt_PT.

Reply:
I remember this add-on and know this is pt_PT. You can provide a new entry with pt_BR. Or simply ditch the pt_PT if the translation is 100% equivalent